_posts/2018-05-16-cockpit-168.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Cockpit 168
+author: pitti
+date: 2018-05-16 11:00
+tags: cockpit linux
+slug: cockpit-168
+category: release
+summary: Cockpit with improved root privilege check
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.
+Here are the release notes from version 168.
+
+### Improve checks for root privilege availability
+
+Many actions, like rebooting the machine or joining an IPA domain, can only be
+done by an administrator (`root`). To decide whether to enable these actions,
+Cockpit previously checked if the user was a member of the `wheel`
+(RHEL/Fedora) or `sudo` (Debian) groups.
```

Comment:
Many actions, like rebooting or joining an IPA domain, can only be performed by
an administrator (aka: `root`).

*[IPA]: identity management system ("Identity, Policy, Audit")
*[aka]: also known as

Reply:
This sounds a bit ambiguous, as one does not "reboot an IPA domain". One could flip it around: "Many actions, like joining an IPA domain or rebooting, can only..."